spec/services/sign_in/token_serializer_spec.rb

```diff
@@ -128,6 +128,30 @@
           expect(subject).to eq({})
         end
       end
+
+      context 'when client_config has refresh_token_path' do
+        let(:refresh_path) { '/v0/sign_in/test' }
+
+        before do
+          client_config.refresh_token_path = '/v0/sign_in/test'
+        end
+
+        it 'returns the refresh_token_path from client_config' do
+          subject
+          expect(cookies[refresh_token_cookie_name][:path]).to eq(refresh_path)
+        end
+      end
+
+      context 'when client_config does not have refresh_token_path' do
+        before do
+          client_config.refresh_token_path = nil
+        end
+
+        it 'returns default refresh_token when path is nil' do
+          subject
+          expect(cookies[refresh_token_cookie_name][:path]).to eq(refresh_path)
+        end
+      end
```

Comment:
Up near the top of this file there is a `client_config` that is defined:
```ruby
let(:client_config) { create(:client_config, authentication:, anti_csrf:) }
```

You should add `refresh_token_path` to this and define a `refresh_token_path`. 
Then in your tests you modify `refresh_token_path` to what you want. You shouldn't be modifying test objects like this: `client_config.refresh_token_path = nil`

You should also modify the `expected_refresh_token_cookie` to work with your newly defined `refresh_token_path`. Then, in your assertions, you should check that the entire cookie equals the expected cookie, not just the path.

Reply:
@rileyanderson  line 97 to 100 is testing the testing the entire cookie. Is there a need to repeat this? I do see a need to replace refresh_path in "expected_refresh_token_cookie"  with refresh_token_path that's added to 
let(:client_config) { create(:client_config, authentication:, anti_csrf:, refresh_token_path:) }

97  it 'sets refresh token cookie' do
        subject
        expect(cookies[refresh_token_cookie_name]).to eq(expected_refresh_token_cookie)
100     end